README.md

```diff
@@ -24,14 +24,15 @@ For non Sublime development a bunch of other IDE plugins are available [here](ht
 ## Log style
 To try and keep consistency across our systems
 type and namespace
-query    -> payload
-reply    -> payload
-request  -> payload
-response -> payload
-log      -> system
-responseTime -> system
-system   -> error
-thirdPartySystem -> error
+
+		query    -> payload
```

Comment:
Look at the raw. https://raw.githubusercontent.com/holidayextras/loggly-interface/fix-docs/README.md
I think you would agree as unformatted raw markdown it is easier to read with indentation.